packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/server.ts

```diff
@@ -21,9 +21,9 @@ export function getServerConfig(wco: WebpackConfigOptions) {
     const { scripts, styles, hidden } = wco.buildOptions.sourceMap;
 
     extraPlugins.push(getSourceMapDevTool(
-      scripts,
-      styles,
-      hidden,
+      scripts === undefined ? true : scripts,
+      styles === undefined ? true : styles,
+      hidden === undefined ? true : hidden,
```

Comment:
When undefined` it's `true`? this is changing the behaviour

Reply:
Current behavior is to default to true, no? The schema says it's true by default, the type says it's optional.

Reply:
It’s true for souceMap but not when it’s a complex object, when it is a complex object at the moment, hidden is also false.